lms/djangoapps/mobile_api/users/views.py

```diff
@@ -60,7 +60,7 @@ class UserDetail(generics.RetrieveAPIView):
     """
     queryset = (
         User.objects.all()
-        .select_related('profile', 'course_enrollments')
+        .select_related('profile')
```

Comment:
If you look at the UserSerializer here:
https://github.com/edx/edx-platform/blob/django-upgrade/1.8/lms/djangoapps/mobile_api/users/serializers.py#L104-L107

You will see that the course_enrollments that is sent back via the JSON is a HyperlinkedIdentityField which does not return any of the enrolled courses data. It simply builds a link to the courseenrollment-detail end point using the username from the user query.